compiler/qsc_eval/src/state/tests.rs

```diff
@@ -647,9 +647,9 @@ fn check_get_latex_for_real() {
     );
     assert_latex_for_real(
         &expect!([r#"
-            "0.00025"
+            "0.0025"
         "#]),
-        1.0 / 4000.0,
+        1.0 / 400.0,
```

Comment:
Why do you change the value used in the test itself, not its output? The number 1/4000 should also have a chance to be printed

Reply:
Done. Now this would be 0.0003 because of truncation to 4 decimal points. In general, I am not a big fan of testing with decimals, especially non-terminating.  I have seen weird test failures(passes on one, fails on other) due to slight change in decimal accuracy even with tolerances :-(